tests/test_late_interaction_multimodal.py

```
@@ -44,43 +67,68 @@
+            cache[lowercase_model_name] = LateInteractionMultimodalEmbedding(lowercase_model_name)
+        yield cache[lowercase_model_name]
+        if lowercase_model_name not in MODELS_TO_CACHE:
+            model_inst = cache.pop(lowercase_model_name)
+            if is_ci:
+                delete_model_cache(model_inst.model._model_dir)
+            del model_inst
+
+    yield get_model
 
+    if is_ci:
+        for name, model in cache.items():
```

Ruff (0.14.10)

Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)